app/src/main/java/ca/bc/gov/bchealth/di/DataStoreModule.kt

```diff
@@ -20,5 +24,25 @@ class DataStoreModule {
 
     @Provides
     @Singleton
-    fun providesDataStore(@ApplicationContext context: Context) = DataStoreRepo(context = context)
+    fun providesDataStore(sharedPreferences: SharedPreferences) =
+        DataStoreRepo(sharedPreferences)
+
+    @Provides
+    @Singleton
+    fun provideEncryptedPreferences(@ApplicationContext context: Context): SharedPreferences {
+
+        // Step 1: Create or retrieve the Master Key for encryption/decryption
+        val masterKey = MasterKey.Builder(context, MasterKey.DEFAULT_MASTER_KEY_ALIAS)
```

Comment:
MasterKey.DEFAULT_MASTER_KEY_ALIAS) is optional, it used this by default

Reply:
Alias is removed.